sdk/cosmos/azure-cosmos/docs/Metrics.md

```diff
@@ -0,0 +1,174 @@
+| cosmos.client.op.actualItemCount | #                                                         | None                   | For feed operations (query, readAll, readMany, change feed) this meter capture the actual item count in responses from the service |
+
+### Metrics for requests to the Cosmos DB Gateway endpoint
+
+| Name                              | Unit     | Percentiles            | Description                                                |
+| --------------------------------- | -------- | ---------------------- | ---------------------------------------------------------- |
+| cosmos.client.req.gw.latency      | duration | 95th, 99th + histogram | End-to-end duration spent for processing the request       |
+| cosmos.client.req.gw.timeline.xxx | duration | 95th, 99th + histogram | Duration spent in different stages of the request pipeline |
+| cosmos.client.req.reqPayloadSize  | bytes    | None                   | The request payload size in bytes                          |
+| cosmos.client.req.rspPayloadSize  | bytes    | None                   | The response payload size in bytes                         |
+
+### Metrics for communication with the Cosmos DB backend replicas via direct TCP (aka RNTBD)
+
+| Name                                                         | Unit     | Percentiles            | Description                                                  |
+| ------------------------------------------------------------ | -------- | ---------------------- | ------------------------------------------------------------ |
+| cosmos.client.req.rntbd.latency                   | duration | 95th, 99th + histogram | End-to-end duration spent for processing the request                                                       |
+| cosmos.client.req.rntbd.backendLatency            | duration | 95th, 99th + histogram | Duration spent for processing the request in the Cosmos DB service endpoint (self-attested by backend) |
+| cosmos.client.req.rntbd.timeline.xxx                         | duration | 95th, 99th + histogram | Duration spent in different stages of the request pipeline   |
+| cosmos.client.req.reqPayloadSize                             |    bytes    | None                   | The request payload size in bytes                             |
+| cosmos.client.req.rspPayloadSize                             |    bytes    | None                   | The response payload size in bytes                             |
+| cosmos.client.req.rntbd.addressResolution.latency | duration | 95th, 99th + histogram | Duration spent for resolving physical addresses of replica for a certain partition |
+| cosmos.client.req.rntbd.stats.endpoint.acquiredChannels      | #        | 95th, 99th + histogram | Number of new TCP connections being established per Cosmos DB service endpoint |
+| cosmos.client.req.rntbd.stats.endpoint.availableChannels     | #        | None                   | Number of established TCP connections per  Cosmos DB service endpoint |
```

Comment:
why the unit of availableChannels is different than the acquiredChannels

Reply:
Fair - I removed percentiles on both - min, max, avg per collection interval is sufficient for this one.